lib/mllparty/connection_hub/client_wrapper.ex

```diff
@@ -34,11 +34,14 @@ defmodule MLLParty.ConnectionHub.ClientWrapper do
         start:
           {MLLP.Client, :start_link,
            [ip, port, [use_backoff: false, auto_reconnect_interval: 1000]]}
+      },
+      %{
```

Comment:
Looks like you added `ClientMonitor.client_spec/3` but forgot to replace this map

Reply:
Yes, this could be replaced with the child_spec